manifests/charts/gateway/templates/hpa.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta2
+apiVersion: autoscaling/v2
```

Comment:
This makes the minimum k8s version 1.23, which is way to high for now. We will need to support multiple versions concurrently and detect which to use

Reply:
I think this is still a concern?

Reply:
You might be looking at a older revision? I actually updated it already, can you check again? @howardjohn

Reply:
It seems to be the latest version - there are a lot of templates

Reply:
ah I missed the helm specific one, updated